### PR TITLE
Update testing instructions based on WG feedback

### PR DESCRIPTION
**Pull request by dlazin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +55,16 @@ may be out of date, the source of truth is whether the nearest anchor to a norma
...
 
-1. If you are an owner of [w3c/epub-tests](https://github.com/w3c/epub-tests/), create a branch on
-   that repo for your new test. Otherwise, fork the repo and create a branch on your fork. (It's
-   easier for reviewers to clone a PR to validate the test if it's in the original repo.)
+1. If you are an owner of [w3c/epub-tests](https://github.com/w3c/epub-tests/), create a branch on that repo for your new
+   test. Otherwise, fork the repo and create a branch on your fork. (It's easier for reviewers to clone a PR to validate the
+   test if it's in the original repo.)
 
 1. Within the branch, copy the
    [test template](https://github.com/w3c/epub-tests/tree/dir/tests/reading-systems/XX-Test-Template/epub-template)
```
Comment (iherman) [inline README.md:70]: The URLs for the templates below are outdated; it should be `https://github.com/w3c/epub-tests/tree/main/tests/xx-epub-template` and `https://github.com/w3c/epub-tests/tree/main/tests/xx-fixed-layout-template`

File: README.md [hunk 2]
```diff
@@ -77,6 +83,10 @@ may be out of date, the source of truth is whether the nearest anchor to a norma
    not to meet requirements, but if you cannot find *any* reading system that processes the test as expected, that may
    indicate an implementation mistake in the test. Fix as necessary.
 
+1. Run the `.epub` through the online [EPUB Validator](http://validator.idpf.org/) or the
+   [EPUBCheck](https://www.w3.org/publishing/epubcheck/) command-line tool to ensure you didn't make any silly mistakes. Fix
+   if you did.
+
 1. Create a pull request for your test change, including both the uncompressed folder and the compressed `.epub` file. Please
    ensure the PR's description clearly indicates which statement is being tested. Await review.
```
Comment (iherman) [inline README.md:92]: The URL-s to be added to the data-tests attributes are now different. They should be of the form `#{testid}`

**Comment by iherman** [comment]
The errors I found were not on the changes, hence it was difficult to place the comments at the right place. I hope it is all right nevertheless.

**Comment by iherman** [comment]
The metadata section lists some coverage values. Should we add to the instructions that if a new coverage value is to be used, then the readme file should also be changed by the tester as part of the same PR?

File: README.md (hunk 1 above)
Comment (dlazin) [reply README.md]: Oops, thanks! Addressed in https://github.com/w3c/epub-tests/pull/40/commits/2cf59e3cd8f47a43012e699df6c7d5771e3ab616.

**Comment by dlazin** [comment]
> The errors I found were not on the changes, hence it was difficult to place the comments at the right place. I hope it is all right nevertheless.

Yep, all good and clear!

**Comment by dlazin** [comment]
> The metadata section lists some coverage values. Should we add to the instructions that if a new coverage value is to be used, then the readme file should also be changed by the tester as part of the same PR?

Good idea. Also done in https://github.com/w3c/epub-tests/pull/40/commits/2cf59e3cd8f47a43012e699df6c7d5771e3ab616.

**Review by bduga — APPROVED** [review]
These changes look good to me, and seem to capture the feedback from the meeting.

**Pull request merged**